content/cosign-releases-bucket-deprecation.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "Sigstore Announcement: No Longer Publishing Cosign Releases to GCS Bucket"
+date = "2023-07-31"
+tags = ["sigstore","cosign"]
+draft = false
+author = "sigstore TSC"
+type = "post"
++++
+
+We are announcing that we will stop publishing [Cosign](https://github.com/sigstore/cosign) releases to the GCS bucket named `cosign-releases`. The current v2.1.1 release of Cosign is the last release that will be pushed to the bucket, and public access to the GCS bucket will be removed on October 31st, 2023.
```

Comment:
I wonder if a sentence that foreshadows why would be useful. E.g.,

`In order to provide a unified story for cosign releases, we are announcing...`

This is to say, part of me wants the "top of the funnel" to have a way to say "we still had other ways to provide cosign this is just about the bucket"